client/platform/desktop/backend/serializers/viame.ts

```diff
@@ -258,15 +258,21 @@ async function parse(input: Readable, imageMap?: Map<string, number>): Promise<A
     });
     parser.on('readable', () => {
       let record: string[];
+      let hasFilenames;
```

Comment:
This variable is named like a boolean variable, but it's not boolean.  It's true/false/undefined.  Please declare its type here so TS can enforce assignment.